twogplus/certificates.py

```diff
@@ -88,8 +88,7 @@ def assert_cert_sign(cose_data: bytes):
             break
     else:
         raise Exception(
-            "Unable validate certificate signature: "
-            f"kid '{required_kid}' not found"
+            "Unable validate certificate signature: " f"kid '{required_kid}' not found"
```

Comment:
This whole line should be one f-string not two string literals

Reply:
correct (that was my code formatter, I never touched that line myself)
```suggestion
            f"Unable validate certificate signature: kid '{required_kid}' not found"
```